libdrgn/drgn.h.in

```diff
@@ -2566,6 +2566,49 @@ struct drgn_error *drgn_format_type(struct drgn_qualified_type qualified_type,
  * @{
  */
 
+/**
+ * The kind of symbol binding (e.g. local or global), from the ST_BIND ELF
+ * field.
+ */
+enum drgn_symbol_binding {
+	/** Some unknown binding type. */
+	DRGN_SYMBOL_BINDING_UNKNOWN = 0,
+	/** A symbol local to its module. */
+	DRGN_SYMBOL_BINDING_LOCAL,
+	/** A global symbol, accessible to all modules. */
+	DRGN_SYMBOL_BINDING_GLOBAL,
+	/** A weak symbol, which could be overridden. */
+	DRGN_SYMBOL_BINDING_WEAK,
+	/* The above are ordered to match elf.h (offset by 1), do not change. */
```

Comment:
I meant the comment to warn against reordering these, since they're implicitly numbered. `DRGN_SYMBOL_BINDING_UNIQUE` is separate and explicitly numbered so doesn't have a risk. How about I change the comment to read `do not reorder`? (same below)

Reply:
Resolving this, feel free to reopen if you don't like the rewording.